ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -56,8 +56,9 @@ private[kvutils] case class CommitContext(
 
   /** Reads key from input state.  Records the key as being accessed even if it's not available. */
   def read(key: DamlStateKey): Option[DamlStateValue] = {
+    val value = inputs.getOrElse(key, throw Err.MissingInputState(key))
```

Comment:
This will not mark a key as being accessed in case it is not available. Is this your intention? The comment now contradicts this behaviour.

Reply:
Yeah, you're right. I was reading "not available" as "is explicitly `None`", but it probably means "not in the map at all".